src/Frontend/Modules/Faq/Js/Faq.js

```diff
@@ -22,17 +23,19 @@ jsFrontend.faq.feedback =
 		// useful status has been changed
 		$('#usefulY, #usefulN').on('click', function()
 		{
-			// get useful status
-			var useful = ($('#usefulY').attr('checked') ? true : false);
+			// init useful status
+			var useful = false;
```

Comment:
can't you just do `var useful = $("#faqFeedbackForm input[type='radio']:checked").val() === 'Y';`?